server/modules/socket_endpoints/ScreenSocket.js

```diff
@@ -64,10 +64,19 @@ function emitNewPhoto() {
   if (newPhotoInterval === null) {
     newPhotoInterval = setInterval(emitNewPhoto, 10000);
   }
-  fetch(`${process.env.LARAVEL_ENDPOINT}/api/photos/random_photo`)
-    .then((res) => res.json())
-    .then((newPhoto) => {
-      photo = newPhoto;
-      endpoint.emit("photo-update", photo);
-    });
+  try {
+    fetch(`${process.env.LARAVEL_ENDPOINT}/api/photos/random_photo`)
+        .then((res) => res.json())
+        .then((newPhoto) => {
+          photo = newPhoto;
+          endpoint.emit("photo-update", photo);
+        });
+  }catch(e){
```

Comment:
please use the proper error handling technique for the native fetch api by adding a `.catch()` handler and/or checking for `res.ok`. For more info check [the docs](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#checking_that_the_fetch_was_successful).